src/other/broadcasting.jl

```diff
@@ -128,13 +186,17 @@ function Base.copyto!(df::AbstractDataFrame, bc::Base.Broadcast.Broadcasted{<:Ba
         end
         df
     else
-        copyto!(df, convert(Broadcasted{Nothing}, bc))
+        copyto!(df, convert(Base.Broadcast.Broadcasted{Nothing}, bc))
     end
 end
 
+Base.Broadcast.broadcast_unalias(dest::DataFrameRow, src) =
+    Base.Broadcast.broadcast_unalias(parent(dest), src)
+
 function Base.copyto!(dfr::DataFrameRow, bc::Base.Broadcast.Broadcasted)
-    for I in eachindex(bc)
-        dfr[I] = bc[I]
+    bc′ = Base.Broadcast.preprocess(dfr, bc)
+    for I in eachindex(bc′)
```

Comment:
Would `@inbounds` be safe here?

Reply:
It should, but the design is so complex, that I would leave it as is for 6 months to make sure it is well tested. The main reason is that we have to do dynamic dispatch here anyway so `@inbounds` will save very little anyway.